lib/dai-plugin-mcd/src/CdpManager.js

```diff
@@ -34,6 +36,21 @@ export default class CdpManager extends LocalService {
     return new ManagedCdp(id, ilk, this);
   }
 
+  //todo: add in caching here??
+  async getDebtAllCdps(proxyAddress, descending = true) {
+    const ids = await this.getCdpIds(proxyAddress, descending);
+    const debts = await Promise.all(
+      ids.map(c => {
+        const cdp = new ManagedCdp(c.id, c.ilk, this);
+        return cdp.getDebtValue();
+      })
+    );
+    const allDebt = debts.reduce((total, debt) => {
```

Comment:
nice use of reduce. this could be a tidy one-liner: `return debts.reduce((a, b) => a.plus(b))`